Client Support/Renewal/2020-09-02/System/itemInfo_EN.lua

```diff
@@ -232699,7 +232680,8 @@ tbl = {
 			"MATK +20",
 			"Every Refine Increases MATK.",
```

Comment:
If you fix something, please check other lines as well and replace this as well with:
```
"^0000FFFor each Refine Level^000000:",
"MATK + 1",
```